packages/fxa-settings/src/components/FormPasswordWithInlineCriteria/index.stories.tsx

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Subject } from './mocks';
+import AppLayout from '../AppLayout';
+import FormPasswordInline from '.';
+import { Meta } from '@storybook/react';
+import { withLocalization } from 'fxa-react/lib/storybooks';
+
+export default {
+  title: 'Components/FormPasswordWithInlineCriteria',
+  component: FormPasswordInline,
+  decorators: [withLocalization],
+} as Meta;
+
+export const ResetPassword = () => (
```

Comment:
Is it possible to show the error state of passwords not matching in storybook?

Reply:
not sure how to do this since we don't pass an initial password state into the component. you can see it if you type something in though.

Reply:
This now works well to test in Storybook!